packages/contracts/src/manager/libraries/LibAppStorage.sol

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {LibDiamond} from "./LibDiamond.sol";
+import "solidstate/contracts/access/access_control/AccessControl.sol";
+import { AccessControlStorage } from "solidstate/contracts/access/access_control/AccessControlStorage.sol";
```

Comment:
> we are only using that import for a constant
> 
> ```
>  bytes32 internal constant DEFAULT_ADMIN_ROLE = 0x00;
> ```
> 
> we would better add that constant with the others instead of importing a whole class

my mistake we also use it for `Modifiers` inherits `AccessControl`

Reply:
> we would better add that constant with the others instead of importing a whole class

Yeah, make sense